lib/honeybadger.js

```diff
@@ -55,13 +56,14 @@ function Honeybadger(opts) {
 }
 
 function withUncaught() {
-  if (!uncaughtExceptionInstalled) {
+  if (this.reportUncaught !== false && !uncaughtExceptionInstalled) {
     var self = this;
-    process.on('uncaughtException', function(err) {
+    uncaughtExceptionHandler = function (err) {
       self.notify(err, function() {
         self.onUncaughtException(err);
       });
-    });
+    }
+    process.on('uncaughtException', uncaughtExceptionHandler);
     uncaughtExceptionInstalled = true;
```

Comment:
We could just move this out of the if block and set it true when it gets called first, that way we don't have to worry about future clients. Or, like mentioned in #30 we could just set the value of `uncaughtExceptionInstalled` to true in `configure` if `reportUncaught` is false